docs/masks.rst

```diff
@@ -160,6 +161,7 @@ as well as a cutout with the data multiplied by the mask::
     >>> mask = aperture.to_mask(mode='exact')
     >>> data = mask.cutout(hdu.data)
     >>> weighted_data = mask.multiply(hdu.data)
+    >>> pf.close()
```

Comment:
I closed some dangling file pointers. These don't fix any errors but I think it is good practice anyway.